Main.hs

```diff
@@ -63,53 +64,10 @@ import Agda.Utils.Impossible
 import Agda.Utils.Maybe.Strict (toLazy, toStrict)
 import Agda.Utils.Monad
 import Agda.Utils.Size
+import Agda.Utils.Functor
 
 import HsUtils
 
-multilineText :: Monad m => String -> m Doc
-multilineText s = vcat $ map text $ lines s
-
--- tracing
```

Comment:
is it intended to delete this stuff?

Reply:
We currently use it to print an error when the default implementations across records do not match, so I guess we keep it for now.